doc/integrator/api.rst

```diff
@@ -6,4 +6,18 @@ JavaScript API
 Any c2cgeoportal application exposes a JavaScript API. This API can be used in third-party applications,
 like CMS applications.
 
-.. TODO: fill for new API
+In the folder ``geoportal/<package>_geoportal/static-ngeo/api/apihelp/`` we have all the files related to
```

Comment:
Suggestion:
The API help is in the folder ``geoportal/<package>_geoportal/static-ngeo/api/apihelp/``.